frontend/src/app/core/helpers/time-zone-helper.ts

```diff
@@ -1,30 +1,6 @@
 import { ITimeZone } from '@core/models/ITimeZone';
 
-export const getPossibleTimeZones = (): ITimeZone[] => [
-    { value: -660, displayValue: 'GMT-11' },
-    { value: -600, displayValue: 'GMT-10' },
-    { value: -540, displayValue: 'GMT-9' },
-    { value: -480, displayValue: 'GMT-8' },
-    { value: -420, displayValue: 'GMT-7' },
-    { value: -360, displayValue: 'GMT-6' },
-    { value: -300, displayValue: 'GMT-5' },
-    { value: -240, displayValue: 'GMT-4' },
-    { value: -180, displayValue: 'GMT-3' },
-    { value: -120, displayValue: 'GMT-2' },
-    { value: -60, displayValue: 'GMT-1' },
-    { value: 0, displayValue: 'GMT' },
-    { value: 60, displayValue: 'GMT+1' },
-    { value: 120, displayValue: 'GMT+2' },
-    { value: 180, displayValue: 'GMT+3' },
-    { value: 240, displayValue: 'GMT+4' },
-    { value: 300, displayValue: 'GMT+5' },
-    { value: 360, displayValue: 'GMT+6' },
-    { value: 420, displayValue: 'GMT+7' },
-    { value: 480, displayValue: 'GMT+8' },
-    { value: 540, displayValue: 'GMT+9' },
-    { value: 600, displayValue: 'GMT+10' },
-    { value: 660, displayValue: 'GMT+11' },
-    { value: 720, displayValue: 'GMT+12' },
-    { value: 780, displayValue: 'GMT+13' },
-    { value: 840, displayValue: 'GMT+14' },
-];
+export const TimeZoneFullNameMapper = (fullTimeZoneName: string): ITimeZone => ({
+    nameValue: fullTimeZoneName.substring(0, fullTimeZoneName.length - 9),
+    timeValue: fullTimeZoneName.substring(fullTimeZoneName.length - 7, fullTimeZoneName.length - 1),
+});
```

Comment:
Why do we need this
As I remember library has separate values for these items

Reply:
We need this because we save in our db full time zone name and time value in minutes. And when we get data from db (full time zone) then we divide it by 2 values for showing in combo box

Reply:
I think we can store in db two separate fields

Reply:
Obviously we store in db two separate fields. Please read again my comment or write more widely your comment

Reply:
Okay, then return in response two separate fields?

Reply:
I still think that we don't need this